truss-chains/examples/audio-transcription/transcribe.py

```diff
@@ -12,8 +12,11 @@
     apt_requirements=["ffmpeg"],
     pip_requirements=["google-auth", "google-cloud-bigquery"],
 )
+
+
 # Whisper is deployed as a separate chainlet `whisper_chainlet.py`.
-_WHISPER_URL = "https://model-5wo86gg3.api.baseten.co/development/predict"
+MODEL_ID = ""  # Fill this in with the model id of the deployed whisper chainlet
```

Comment:
Maybe add an assert with a helpful message that prevents the deploy in case this stays empty so that the user doesn't get a confusing error down the line.

Reply:
<img width="1042" alt="Cursor_and_transcribe_py_—_truss__Codespaces__zany_happiness_" src="https://github.com/basetenlabs/truss/assets/850115/05d88c12-972f-4398-bca4-7bdd3e249257">